fetch.bs

```diff
@@ -1084,10 +1085,10 @@ Unless stated otherwise, it is "<code>basic</code>".
 <hr>
 
 <p>A <dfn export>subresource request</dfn> is a <a for=/>request</a>
-whose <a for=request>destination</a> is "<code>font</code>",
-"<code>image</code>", "<code>manifest</code>", "<code>media</code>",
-"<code>script</code>", "<code>style</code>", "<code>xslt</code>", or the empty
-string.
+whose <a for=request>destination</a> is "<code>fetch</code>",
+"<code>font</code>", "<code>image</code>", "<code>manifest</code>",
+"<code>media</code>", "<code>script</code>", "<code>style</code>",
+"<code>xslt</code>", or the empty string.
```

Comment:
I think the main point is to have it here in case someone else needs to use it too. Just like Fetch defines header value normalization and not XMLHttpRequest.